main/FatesHistoryInterfaceMod.F90

```diff
@@ -6905,11 +6951,17 @@ subroutine define_history_vars(this, initialize_variables)
          index=ih_hydraulicmortality_carbonflux_si_pft)
 
     call this%set_history_var(vname='FATES_MORTALITY_CSTARV_CFLUX_PF', units='kg m-2 s-1',    &
-         long='PFT-level flux of biomass carbon from live to dead pool from carbon starvation mortality', &
+         long='PFT-level flux of biomass carbon from live to dead pool from carbon starvation mortality (both continuous and termination)', &
          use_default='active', avgflag='A', vtype=site_pft_r8, hlms='CLM:ALM', &
          upfreq=1, ivar=ivar, initialize=initialize_variables,                 &
          index=ih_cstarvmortality_carbonflux_si_pft)
 
+    call this%set_history_var(vname='FATES_MORTALITY_CSTARV_CONT_CFLUX_PF', units='kg m-2 s-1',    &
```

Comment:
This variable name might get cropped when its passed to the HLM history output machinery since its so long

Reply:
I shortened it to just under 32 characters